src/flow.h

```diff
@@ -302,6 +302,40 @@ typedef struct FlowCnf_
     SC_ATOMIC_DECLARE(uint64_t, memcap);
 } FlowConfig;
 
+#if defined(ENABLE_ETM)
+
+#define FLOW_SPLT_MAX_COUNT       32
+#define FLOW_SPLT_MAX_BD_COUNT    65536
+#define FLOW_SPLT_TOCLIENT 0
+#define FLOW_SPLT_TOSERVER 1
+#define FLOW_SPLT_MAX_MSEC 262144  // 2^18
+#define FLOW_SPLT_MAX_LEN 8192    // 2^13
+#define FLOW_SPLT_BD_SIZE 256
+typedef struct PacketSequence_
+{
+    uint32_t dir: 1;
+    uint32_t len: 13;
+    uint32_t delta: 18;
+} PacketSequence;
+
+typedef struct FlowSPLT_
+{
+    uint64_t first_epoch_msec;
```

Comment:
this seems to be set to the first packet in the `FLOW_STATE_ESTABLISHED`, so this will be when TCP state is ESTABLISHED or in other IP protos when we see the first packet in the toclient direction. Since this is not the same as `Flow::startts`, it seems to make sense to have it as a separate field.